src/get_webhook_data.py

```
@@ -158,11 +169,56 @@ async def webhook(r: InputRequest):
+            print(f"# {dt.datetime.utcnow()}: Failed CSV Update Attempt {counter}")
+            counter += 1
+            time.sleep(30)
+    
     if has_updated_github:
         print(f"# {dt.datetime.utcnow()}: Pine Updated on Github")
         print(data_df)
     else:
         print(f"# {dt.datetime.utcnow()}: Failed to update Pine on Github !!")
+    
+    if not has_updated_csv:
+        print(f"# {dt.datetime.utcnow()}: Failed to upload CSV to Github !!")
```

Ruff (0.8.2)

`datetime.datetime.utcnow()` used

(DTZ003)